monitor/monitor_start.py

```diff
@@ -0,0 +1,49 @@
+from monitor import MONITOR_DIR, MONITOR_CONF, MONITOR_DEFAULT_CONF
+from monitor.data_loader import DataLoader
+from monitor.visualizer import Visualizer
+
+from common.config_loader import ConfigLoader
+from common.logger import Logger
+
+from teos import DATA_DIR, DEFAULT_CONF, CONF_FILE_NAME
+
+LOG_PREFIX = "Main"
+logger = Logger(actor="System Monitor Main", log_name_prefix=LOG_PREFIX)
+
+def main(command_line_conf):
+    logger.info("Setting up the system monitor.")
+
+    # Pull in Teos's config file to retrieve some of the data we need.
+    conf_loader = ConfigLoader(DATA_DIR, CONF_FILE_NAME, DEFAULT_CONF, command_line_conf)
+    conf = conf_loader.build_config()
+
+    max_users = conf.get("DEFAULT_SLOTS")
```

Comment:
Not completely sure what max_users is used for, but you're using `DEFAULT_SLOTS`, which is used to define the default number of slots a users gets for a free subscription.

Reply:
Ohhhhhhh I see. Lol, I'm not sure how I reached this conclusion, but I was thinking that was the maximum number of users a watchtower supported. But I see, there's a maximum # of appointments, but not users. Will rework the graph I made with that data bit based on that